homeworks/pontazsolt/3_conditionals_loops/rock_paper_scissors.py

```diff
@@ -0,0 +1,60 @@
+
+# Adjuk meg hány kört akarunk játszani, figyeljünk oda a végeredmény nem lehet döntetlen
+while True:
+    try:
+        count_rounds = int(input("How many turns would you like to play? Write an odd number: "))
+        if count_rounds % 2 == 0:
+            print(f"Must be a winner at the end! Give an odd number:")
+        else:
+            break
+    except ValueError:
+        print("Have to write correct number!")
+
+
+# Pontszámok
+player1_score = 0
+player2_score = 0
+
+#Körök iterálása
+for round in range(count_rounds):
+    print(f"\n{round + 1}. kör: ")
+
+    #Első játékos köre:
+    while True:
+        player1_choice = input("Player 1 turn: 'rock', 'paper' or 'scissors'?: " ).lower()
+        if player1_choice in ["rock", "paper", "scissors"]:
+            break
+        else:
+            print(f"Choose one of them: 'rock', 'paper', 'scissors': ")
+
+    #Második játékos köre:
+    while True:
+        player2_choice = input("Player 2 turn: 'rock', 'paper' or 'scissors'?: " ).lower()
+        if player2_choice in ["rock", "paper", "scissors"]:
+            break
+        else:
+            print("Choose one of them: 'rock', 'paper', 'scissors': ")
+
+    #Eredmény meghatározása:
+    if player1_choice == player2_choice:
+        print("The result is a draw")
```

Comment:
Szuper lenne ha nem tudnának döntetlent játszani egy körben, és ehhez elég lenne egy apró változtatás:

```
current_round = 1
#Körök iterálása
#for round in range(count_rounds):
while current_round <= count_rounds:
    print(f"\n{current_round}. kör: ")

    #Első játékos köre:
    while True:
        player1_choice = input("Player 1 turn: 'rock', 'paper' or 'scissors'?: " ).lower()
        if player1_choice in ["rock", "paper", "scissors"]:
            break
        else:
            print(f"Choose one of them: 'rock', 'paper', 'scissors': ")

    #Második játékos köre:
    while True:
        player2_choice = input("Player 2 turn: 'rock', 'paper' or 'scissors'?: " ).lower()
        if player2_choice in ["rock", "paper", "scissors"]:
            break
        else:
            print("Choose one of them: 'rock', 'paper', 'scissors': ")

    #Eredmény meghatározása:
    if player1_choice == player2_choice:
        print("The result is a draw, try again!")
        continue
    elif (player1_choice == 'rock' and player2_choice == 'scissors') or \
         (player1_choice == 'scissors' and player2_choice == 'paper') or \
         (player1_choice == 'paper' and player2_choice == 'rock'):
        print("Player 1 won that round")
        player1_score += 1
    else:
        print("Player 2 won that round")
        player2_score += 1
    
    current_round += 1
```

Ami it történik:
bevezettünk egy current_round változót, és a for ciklust lecseréltük egy while ciklusra. Ha döntetlen az aktuális kör, a continue-val addig fogja újrakezdeni az aktuális kört, amíg eltérő értéket nem adnak meg (van a conditional végén egy current_round += 1, ez nem fut le, ha a continue meghívódik. Így csak akkor megyünk a következő körre ha nem döntetlen az aktuális kör eredménye.